app/pages/reservation/reservation-information/InternalReservationForm.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import Form from 'react-bootstrap/lib/Form';
+import Row from 'react-bootstrap/lib/Row';
+import Col from 'react-bootstrap/lib/Col';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import injectT from '../../../i18n/injectT';
+import FormTypes from '../../../constants/FormTypes';
+
+export const maxLengths = {
+  internalReservationComments: 1500
+};
+
+function mapStateToProps(state) {
+  return { stateForm: state.form };
+}
+
+class UnconnectedInternalReservationForm extends Component {
+  state = {
+    internalReservationDefaultChecked: true,
+    markAsClosedDefaultChecked: false
+  }
+
+  componentDidMount() {
+    this.props.dispatch(this.internalReservationFormInitialValue());
+  }
+
+  internalReservationFormInitialValue() {
+    return {
+      type: '@@redux-form/CHANGE',
+      meta: {
+        form: 'RESERVATION',
+        field: 'internalReservation',
+        touch: false,
+        persistentSubmitErrors: false
+      },
+      payload: this.state.internalReservationDefaultChecked
+    };
+  }
+
+  render() {
+    const {
+      t,
+      stateForm
+    } = this.props;
+    let internalReservationCommentsLength = 0;
+    if (
+      stateForm
+      && stateForm.RESERVATION
+      && stateForm.RESERVATION.values
+      && stateForm.RESERVATION.values.internalReservationComments
```

Comment:
lodash.get

Reply:
I don't get how to use it in this case?
If I do something like this:
`let internalReservationCommentsLength = get(stateForm, stateForm.RESERVATION.values.internalReservationComments.length, 0);`
It gives me an error: `Uncaught TypeError: Cannot read property 'values' of undefined`
So, could you help me a little bit?

Reply:
Fixed that